File: get_cpu_temp.py
#!/usr/bin/python -u

import sys, os, re

def execute_command():
	cmd='sensors'
    
	return os.popen(cmd).read().split('\n')

def main():
	
	core_temp_regex = re.compile('^(Core \d:)\s+[+-]?(\d+)\.\d+\s*C')

	data_list = execute_command()

	for line in data_list:
		if core_temp_regex.match(line):
			formatted = re.sub('\s','',core_temp_regex.match(line).group(1)) + core_temp_regex.match(line).group(2)
			print formatted,

if __name__ == "__main__":

        main()

File: get_cpu_temp.pl
#!/usr/bin/env perl
use strict;
use warnings;

my $cmd = "sensors";    
my @output = `$cmd`;    
chomp @output;

foreach my $line (@output)
{
	if($line =~ m/^(Core) (\d):\s+[+-]?(\d+)\.\d+\s*C/){

		print "$1:".$3;

	}

}
